zenodo_uploader.py

```diff
@@ -1,25 +1,48 @@
 import os
-from pathlib import Path
-import yaml
-from pyzenodo3.upload import upload
+
 import argparse
+import requests
+import traceback
+
 
-def upload_to_zenodo(file_name_to_upload, base_url="https://sandbox.zenodo.org/api/"):
-    upload(
-        datafn=Path(file_name_to_upload),
-        token=os.getenv("ZENODO_ACCESS_TOKEN", ""),
-        metafn="",
+def upload_to_zenodo(file_name_to_upload, zenodo_server="https://sandbox.zenodo.org/api/deposit/depositions"):
+    headers = {"Content-Type": "application/json"}
+    params = {"access_token": os.getenv("ZENODO_ACCESS_TOKEN", "")}
+    r = requests.post(
+        zenodo_server,
+        params=params,
+        json={},
+        headers=headers,
     )
+    deposition_id = r.json()["record_id"]
+    #print(r.status_code)
+    #print(r.json())
+    bucket_url = r.json()["links"]["bucket"]
+    filename = file_name_to_upload
```

Comment:
Probably we want to always convert the path to the absolute path, and check its existence.
```suggestion
    filename = os.path.abspath(file_name_to_upload)
    if not os.path.isfile(filename):
        raise FileNotFoundError(f"The file, specified for uploading does not exist: {filename}")
```